.github/workflows/create_release.yml

```diff
@@ -24,13 +24,13 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: 'zulu'  # zulu as it supports a wide version range
-          java-version: '11'  # earliest LTS and last that can compile the 1.6 release profile.
+          java-version: '17'  # earliest LTS and last that can compile the 1.6 release profile.
```

Comment:
I guess this can't compile 1.6 profile

Reply:
yeah 11 was the last that could

Reply:
for posterity! https://bugs.openjdk.org/browse/JDK-8206440